forest/sycamore/base.py

```diff
@@ -122,7 +122,7 @@ def compute_survey_stats(
                     os.path.join(output_folder, "summaries",
                                  "submits_summary.csv"), index=False
                 )
-                if Frequency(submits_timeframe) == Frequency.BOTH:
+                if Frequency(submits_timeframe) == Frequency.HOURLY_AND_DAILY:
```

Comment:
The value of `submits_timeframe` in this function's signature and its docs should be updated too.

Also, do you think it'd be useful to switch its type from string to enum like in `get_submits_for_tableau()` below?